pkg/plugin/installer/base.go

```diff
@@ -35,5 +36,9 @@ func (b *base) Path() string {
 	if b.Source == "" {
 		return ""
 	}
+	helmPluginsDir := os.Getenv("HELM_PLUGINS")
```

Comment:
Two things....

1. The plugins directory should be passed in rather than directly pulled from the environment variable. The could be a public property on `base` and set as part of the install command.
2. There are other places this needs to be modified beyond base. I didn't look entirely but I did see http installer has a `Path` function that overrides this one. It needs to be modified, too.

Reply:
I will fix it.